TypeError: _(...).sortByOrder is not a function

```
❯ WS_SECRET="asdfasdfasdf" npm start

> netstats-server@0.1.0 start /home/user/.opt/netstats-server
> node ./bin/www

/home/user/.opt/netstats-server/lib/history.js:595
			.sortByOrder( 'height', false )
			 ^

TypeError: _(...).sortByOrder is not a function
    at History.getCharts (/home/user/.opt/netstats-server/lib/history.js:595:5)
    at Collection._debounced._.debounce.leading (/home/user/.opt/netstats-server/lib/collection.js:272:21)
    at invokeFunc (/home/user/.opt/netstats-server/node_modules/lodash/lodash.js:10335:23)
    at trailingEdge (/home/user/.opt/netstats-server/node_modules/lodash/lodash.js:10384:18)
    at Timeout.timerExpired [as _onTimeout] (/home/user/.opt/netstats-server/node_modules/lodash/lodash.js:10372:18)
    at ontimeout (timers.js:380:14)
    at tryOnTimeout (timers.js:244:5)
    at Timer.listOnTimeout (timers.js:214:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! netstats-server@0.1.0 start: `node ./bin/www`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the netstats-server@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   "dependencies": {
     "body-parser": "^1.18.3",
     "chalk": "^2.4.1",
-    "d3": "^5.7.0",
+    "d3": "3.5.17",
     "debug": "^4.1.0",
     "express": "^4.16.4",
     "geoip-lite": "^1.3.2",
@@ -27,7 +27,7 @@
     "grunt-contrib-jade": "^1.0.0",
     "grunt-contrib-uglify": "^4.0.0",
     "jade": "^1.11.0",
-    "lodash": "^4.17.11",
+    "lodash": "3.10.1",
     "primus": "^7.3.1",
     "primus-emit": "^1.0.0",
     "primus-spark-latency": "^0.1.1",

